server/api/categoryRouter.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * Router for item actions
+ */
+const Category = require('../db/models/category');
+const express = require('express');
+const Item = require('../db/models/item');
+const router = express();
+router.use(express.json());
```

Comment:
This is required for tests on categoryRouter to work

Reply:
ohh ok!

Reply:
oh actually, if you require index.js and use the full url for categories which is '/category/endpointName' it works, so we don't have to do router.use for each router. 
6bff5ca